### PR TITLE
NNEO-10339 Vendo Gateway API Client

### PR DESCRIPTION
**Pull request by adolfoabegg**

### CODE REVIEW

File: src/Gateway/Request/Details/CreditCard.php [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    /**
+     * Set the expiration month using two digits. Examples: 01 for January, 02 for February, 10 for October, etc.
+     *
+     * @param string $expirationMonth
+     */
+    public function setExpirationMonth(string $expirationMonth): void
+    {
+        if (strlen($expirationMonth) < 2) {
+            $expirationMonth = '0' . $expirationMonth;
```
Comment (zbigniewwiech) [inline src/Gateway/Request/Details/CreditCard.php:52]: $expirationMonth = \substr('0' . $expirationMonth, -2); //one "if" less ;)

File: src/Util/HttpClientTrait.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+     */
+    public function getHttpRequest(
+        string $method,
+        string $uri,
+        array $headers = [],
+        ?string $body = null,
+        string $version = '1.1'
+    ): Request
+    {
+        $headers['X-VENDO_PHP_SDK_VERSION'] = Vendo::SDK_VERSION;
```
Comment (zbigniewwiech) [inline src/Util/HttpClientTrait.php:49]: It will not probably work. Custom headers with underscores are (and should be) silently discarded - security.

**Review by zbigniewwiech — CHANGES_REQUESTED** [review]

File: src/Gateway/Request/Details/CreditCard.php (hunk 1 above)
Comment (adolfoabegg) [reply src/Gateway/Request/Details/CreditCard.php:52]: nice!

File: src/Util/HttpClientTrait.php (hunk 2 above)
Comment (adolfoabegg) [reply src/Util/HttpClientTrait.php:49]: ok, noted.

**Review by zbigniewwiech — APPROVED** [review]

**Pull request merged**